classification/__main__.py

```diff
@@ -46,5 +46,6 @@ def process_data_and_extract_profiles(song_id, song_file, song_output_file):
     pool.close()
 
     print(tabulate(res,
-                   headers=['Song ID', 'Timbre', 'Mood relaxed', 'Mood party'],
+                   headers=['Song ID', 'Timbre', 'Mood relaxed', 'Mood party', 'Mood aggressive', 'Mood happy', 'Mood sad'],
```

Comment:
Try to make this multiple lines. An example of something that is pep8-compliant:
```python
headers=[
    'Song ID', 'Timbre', 'Mood relaxed', 'Mood party', 
    'Mood aggressive', 'Mood happy', 'Mood sad'
    ],
```

Reply:
yeah would probably make this 3 lines